en/02_Developer_Guides/20_Deprecations/index.md

```diff
@@ -19,7 +19,7 @@ going to be removed and how they need to update their code.
 - Add a `@deprecated` item to the docblock tag, with the version when the code was deprecated and a message indicating what to use instead.
 - Update the deprecated code to throw a [Deprecation::notice()](api:SilverStripe\Dev\Deprecation::notice()) warning.
 - Both the docblock and error message should contain the **version** where the functionality is deprecated from.
-  So, if you're committing the change to a *4.12* minor release, the version will be *4.12.0*.
+  So, if you're committing the change to a *1.2* minor release, the version will be *1.2.0*.
```

Comment:
Use `1.2`  since that's what's used in some of the examples.
Updated the examples so _all_ examples use this as well.